README.md

```diff
@@ -1,2 +1,45 @@
 # hexgrid-react
 A library to render objects using hexagonal grid layout.
+
+# :arrow_down:Important
+
+  
+Follow these steps:hammer: for contributing:
+
+:green_heart:Fork the repo.
+
+:pushpin:Create a separate branch for any bug/feature with branch name: <grid/bug>
+
+:white_check_mark:Commit once a program is complete.
```

Comment:
Can you change it to 'Commit once a feature is done/a bug is fixed'.

Reply:
No